catalyst-gateway/tests/Earthfile

```diff
@@ -35,14 +19,16 @@ package-schemathesis:
                                  --force-schema-version=$schema_version --junit-xml /results/junit-report.xml
     SAVE IMAGE schemathesis:$tag
 
-fuzzer-api-test:
-    FROM earthly/dind:alpine
-    COPY integration/docker-compose.yml .
+# test-fuzzer-api - Fuzzy test cat-gateway using openapi specs
+test-fuzzer-api:
+    FROM earthly/dind:alpine-3.19
+    RUN apk update && apk add iptables-legacy # workaround for https://github.com/earthly/earthly/issues/3784
+    COPY st-docker-compose.yml .
     WITH DOCKER \
-        --compose docker-compose.yml \
+        --compose st-docker-compose.yml \
```

Comment:
What does `st` mean in `st-docker-compose.yml`?

Reply:
st = schemathesis. I wanted to be explicit that this docker compose is used by the schematestis test

Reply:
@kukkok3 Why not call the file `schemathesis-docker-compose.yml` then?
Over time it may be difficult to reason about `st.`